src/App/Pages/Send/SendAddEditPage.xaml.cs

```diff
@@ -0,0 +1,229 @@
+using System;
+using System.Collections.Generic;
+using Bit.App.Resources;
+using Bit.Core.Abstractions;
+using Bit.Core.Enums;
+using Bit.Core.Utilities;
+using Xamarin.Forms;
+using Xamarin.Forms.PlatformConfiguration;
+using Xamarin.Forms.PlatformConfiguration.iOSSpecific;
+using Entry = Xamarin.Forms.Entry;
+
+namespace Bit.App.Pages
+{
+    public partial class SendAddEditPage : BaseContentPage
+    {
+        private readonly IBroadcasterService _broadcasterService;
+        
+        private SendAddEditPageViewModel _vm;
+        
+        public SendAddEditPage(
+            string sendId = null, 
+            SendType? type = null)
+        {
+            _broadcasterService = ServiceContainer.Resolve<IBroadcasterService>("broadcasterService");
+            InitializeComponent();
+            _vm = BindingContext as SendAddEditPageViewModel;
+            _vm.Page = this;
+            _vm.SendId = sendId;
+            _vm.Type = type;
+            _vm.Init();
+            if (Device.RuntimePlatform == Device.Android)
+            {
+                if (_vm.EditMode)
+                {
+                    ToolbarItems.Add(_removePassword);
+                    ToolbarItems.Add(_copyLink);
+                    ToolbarItems.Add(_shareLink);
+                    ToolbarItems.Add(_deleteItem);
+                }
+                _vm.EditorMargins = new Thickness(0,5,0,0);
+            }
+            if (Device.RuntimePlatform == Device.iOS)
+            {
+                ToolbarItems.Add(_closeItem);
+                if (_vm.EditMode)
+                {
+                    ToolbarItems.Add(_moreItem);
+                }
+                _vm.ShowEditorSeparators = true;
+                _vm.EditorMargins = new Thickness(0,10,0,5);
+                _typePicker.On<iOS>().SetUpdateMode(UpdateMode.WhenFinished);
+                _deletionDateTypePicker.On<iOS>().SetUpdateMode(UpdateMode.WhenFinished);
+                _expirationDateTypePicker.On<iOS>().SetUpdateMode(UpdateMode.WhenFinished);
+            }
+            
+            _typePicker.ItemDisplayBinding = new Binding("Key");
+            _deletionDateTypePicker.ItemDisplayBinding = new Binding("Key");
+            _expirationDateTypePicker.ItemDisplayBinding = new Binding("Key");
+
+            if (_vm.IsText)
+            {
+                _nameEntry.ReturnType = ReturnType.Next;
+                _nameEntry.ReturnCommand = new Command(() => _textEditor.Focus());
+            }
+        }
+        
+        protected override async void OnAppearing()
+        {
+            base.OnAppearing();
+            _broadcasterService.Subscribe(nameof(SendAddEditPage), (message) =>
```

Comment:
You can (and should) remove the `()` around the `(message)` arrow parameter, `message =>`.

Reply:
fixed